### PR TITLE
Add support for multiple Android accounts.

### PR DESCRIPTION
**Pull request by corbinlc**
**Describe the pull request**

Makes it so extraction, running the server and connecting the client all work on a 2nd account.
Still some oddity when running say `sudo apt-get update` but I think that is going to take some PRoot debugging to fix.

**Link to relevant issues**

#607 #762 probably others

### CODE REVIEW

File: termux-app/terminal-term/src/main/java/com/termux/app/TermuxActivity.java [hunk 1]
```diff
@@ -150,8 +152,8 @@ public void onReceive(Context context, Intent intent) {
 
     void checkForFontAndColors() {
         try {
-            @SuppressLint("SdCardPath") File fontFile = new File("/data/data/tech.ula/files/home/.termux/font.ttf");
-            @SuppressLint("SdCardPath") File colorsFile = new File("/data/data/tech.ula/files/home/.termux/colors.properties");
+            @SuppressLint("SdCardPath") File fontFile = new File(this.getFilesDir().getAbsolutePath() + "/home/.termux/font.ttf");
```
Comment (MatthewTighe) [inline termux-app/terminal-term/src/main/java/com/termux/app/TermuxActivity.java]: These lint suppressions might not be necessary now.

File: termux-app/terminal-term/src/main/java/com/termux/app/TermuxService.java [hunk 2]
```diff
@@ -50,10 +50,10 @@
 
     /** Note that this is a symlink on the Android M preview. */
     @SuppressLint("SdCardPath")
-    public static String FILES_PATH = "/data/data/tech.ula/files";
-    public static final String SUPPORT_PATH = FILES_PATH + "/support/";
-    public static final String PREFIX_PATH = FILES_PATH + "/usr";
-    public static final String HOME_PATH = FILES_PATH + "/home";
+    public String FILES_PATH;
```
Comment (MatthewTighe) [inline termux-app/terminal-term/src/main/java/com/termux/app/TermuxService.java]: Non-static fields should use camelcase naming.

File: app/src/main/java/tech/ula/utils/BusyboxExecutor.kt [hunk 3]
```diff
@@ -154,8 +154,7 @@ class BusyboxWrapper {
 
     // Proot scripts expect CWD to be `applicationFilesDir/<filesystem`
     fun addBusyboxAndProot(command: String): List<String> {
-        val commandWithProot = "support/execInProot.sh $command"
-        return listOf("support/busybox", "sh", "-c", commandWithProot)
+        return listOf("support/busybox", "sh", "support/execInProot.sh") + command.split(" ")
```
Comment (MatthewTighe) [inline app/src/main/java/tech/ula/utils/BusyboxExecutor.kt]: Is splitting the command necessary now?

**Review by MatthewTighe — CHANGES_REQUESTED** [review]

File: app/src/main/java/tech/ula/utils/BusyboxExecutor.kt (hunk 3 above)
Comment (corbinlc) [reply app/src/main/java/tech/ula/utils/BusyboxExecutor.kt]: Yes, the `-c` before treated each arg differently.

**Comment by corbinlc** [comment]
I found and fixed a very old PRoot bug that was manifesting itself when running a 2nd account (no idea why it didn't show up elsewhere).  So, this now should fully work as well as being on the primary account.

**Review by MatthewTighe — APPROVED** [review]

**Pull request merged**